src/DynamoCoreWpf/UI/Themes/Modern/DynamoColorsAndBrushes.xaml

```diff
@@ -80,7 +80,8 @@
 
     <!-- Library text colors-->
     <SolidColorBrush x:Key="MemberButtonText" Color="#aaaaaa" />
-    <SolidColorBrush x:Key="DeafultSearchTextBlockText" Color="#878787" />
+    <SolidColorBrush x:Key="DeafultSearchTextBlockText"
+                     Color="#6D6E71" />
```

Comment:
can you replace the constant colors with named versions like exited previously? maybe we'll reuse these somewhere.